apps/visualization/views.py

```diff
@@ -45,3 +62,237 @@ def get_queryset(self):
+            writer.writerow([
+                item.emergency,
+                item.country_name,
+                item.region,
+                item.iso3,
+                item.admin_level_1,
+                item.indicator_id,
+                item.subvariable,
+                item.indicator_name,
+                item.thematic,
+                item.thematic_description,
+                item.topic,
+                item.topic_description,
+                item.indicator_description,
+                item.type,
+                item.question,
+                item.indicator_value,
+                item.nominator,
+                item.error_margin,
+                item.denominator,
+                item.indicator_month,
+                item.category,
+                item.gender,
+                item.age_group,
+                item.age_info,
+                item.target_group,
+                item.indicator_matching,
+                item.representativeness,
+                item.limitation,
+                item.indicator_comment,
+                item.source_id,
+                item.organisation,
+                item.title,
+                item.details,
+                item.authors,
+                item.methodology,
+                item.sample_size,
+                item.target_pop,
+                item.scale,
+                item.quality_check,
+                item.access_type,
+                item.source_comment,
+                item.publication_channel,
+                item.link,
+                item.source_date,
+                item.sample_type,
+            ])
+        return response
+
+
+class ExportSummaryView(ListAPIView):
+
+    filterset_class = DataCountryLevelPublicFilter
+    queryset = DataCountryLevelPublic.objects.all()
+
+    def get(self, request, format=None):
+        """
+        Export summary data
+        """
+
+        validate_export_params_together(self.request.query_params)
+        filterset = DataCountryLevelPublicFilter(
+            data=self.request.query_params,
+            queryset=self.get_queryset(),
+        )
+        response = HttpResponse(
+            content_type='text/csv',
+            headers={'Content-Disposition': 'attachment; filename="summary.csv"'},
+        )
+        writer = csv.writer(response)
+        writer.writerow([
+            'emergency',
+            'country_name',
+            'iso3',
+            'admin_level_1',
+            'region',
+            'income_group',
+            'fragility_index_fund_for_peace',
+            'population_size',
+            'indicator_id',
+            'subvariable',
+            'indicator_name',
+            'thematic',
+            'thematic_description',
+            'topic',
+            'topic_description',
+            'indicator_description',
+            'type',
+            'indicator_value',
+            'error_margin',
+            'indicator_month',
+            'category',
+        ])
```

Comment:
@bimal125 Instead of hard-code these fields, can we directly get it the table headers/columns (dynamic) ?